lib/vrl/stdlib/src/parse_key_value.rs

```diff
@@ -92,7 +160,14 @@ impl Expression for ParseKeyValueFn {
         let value = self.field_delimiter.resolve(ctx)?;
         let field_delimiter = value.try_bytes_utf8_lossy()?;
 
-        let values = parse(&bytes, &key_value_delimiter, &field_delimiter)?;
+        let whitespace_strict = self.whitespace == Whitespace::Strict;
```

Comment:
Minor point - this seems a bit unnecessary. It doesn't seem any harder for `parse_key_value` to take a `Whitespace` instead of a boolean.